packages/apollo-gateway/src/executeQueryPlan.ts

```diff
@@ -207,7 +208,7 @@ async function executeFetch<TContext>(
 
   let variables = Object.create(null);
   if (fetch.variableUsages) {
-    for (const variableName of Object.keys(fetch.variableUsages)) {
+    for (const variableName of fetch.variableUsages) {
```

Comment:
we changed this from an object whose keys we cared about to just a list of those keys

Reply:
Confirmed.  Seen here above: https://github.com/apollographql/apollo-server/pull/4410/files#diff-a3e1fdf002c7b734a2c6a266edfb4a20L42